chirp/drivers/lt725uv.py

```diff
@@ -1338,7 +1341,8 @@ def make_dtmf(ary, knt):
             """Generate the DTMF code 1-8, NOT a callback."""
             tmp = ""
             if knt > 0 and knt != 0xff:
-                for val in ary[:knt]:
+                for j in range(0, knt, 1):
+                    val = ary[j]
```

Comment:
Can you explain what's going on here? It looks to me like this change should have no effect, and I think the parent radio is already tested as working, so is this just a cosmetic change or functional?

Reply:
I'm still curious about this, especially since it changes a driver that we already think is tested. @KC9HI does this look necessary or beneficial for some reason I'm missing?